evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -1,7 +1,19 @@
 {% load evaluation_filters %}
-
+<div class="d-flex">
+    <h4 class="me-auto">{{ questionnaire_group.0.questionnaire.public_name }}</h4>
```

Comment:
This now does the first questionnaire outside of the loop, I wonder if we could restructure it so that we have the "I can't give feedback" button in the loop, but only included in the first iteration, so instead of
```
<h4>...</h4>
{% if contributor %} ... {% endif %}

{% for form %}
  {% if not forloop.first %} <h4> ... </h4> {% endif %}
  ...
{% endfor %}
```

we would have

```
{% for form %}
  <h4>...</h4>
  {% if forloop.first and contributor %}
    ...
  {% endif ⅜}
{% endfor %}
```

For me, this better captures the intent here and it removes the duplicated "knowledge" about how one questionnaire is layed out in the html (though the last one is rather minor)

Reply:
Yeah, sounds good